app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -102,21 +102,38 @@
             }}
           </p>
         </label>
-
-        <woot-input
-          v-if="greetingEnabled"
-          v-model.trim="greetingMessage"
-          class="medium-9 columns"
-          :label="
-            $t('INBOX_MGMT.ADD.WEBSITE_CHANNEL.CHANNEL_GREETING_MESSAGE.LABEL')
-          "
-          :placeholder="
-            $t(
-              'INBOX_MGMT.ADD.WEBSITE_CHANNEL.CHANNEL_GREETING_MESSAGE.PLACEHOLDER'
-            )
-          "
-        />
-
+        <section v-if="greetingEnabled">
+          <resizable-text-area
+            v-if="textAreaChannels"
+            v-model.trim="greetingMessage"
+            rows="4"
+            type="text"
+            class="medium-9 columns greetings--textarea"
+            :label="
+              $t(
+                'INBOX_MGMT.ADD.WEBSITE_CHANNEL.CHANNEL_GREETING_MESSAGE.LABEL'
+              )
+            "
+            :placeholder="
+              $t(
+                'INBOX_MGMT.ADD.WEBSITE_CHANNEL.CHANNEL_GREETING_MESSAGE.PLACEHOLDER'
+              )
+            "
+          />
+          <label v-else class="greetings--richtext">
+            <woot-message-editor
+              v-model.trim="greetingMessage"
+              :is-format-mode="true"
+              class="input"
+              :placeholder="
+                $t(
+                  'INBOX_MGMT.ADD.WEBSITE_CHANNEL.CHANNEL_GREETING_MESSAGE.LABEL'
+                )
+              "
+              :min-height="4"
+            />
+          </label>
```

Comment:
Can we create a separate component for this? It seems like this logic and styles are repeated in the component below.

Reply:
Moved to a seperate component in https://github.com/chatwoot/chatwoot/pull/2952/commits/e63de4d5bf184a76b255dc03a66e63ccc37f8633